src/crimson/common/interruptible_future.h

```diff
@@ -1179,7 +1179,7 @@ struct interruptor
       std::move(ic),
       std::forward<OpFunc>(opfunc),
       std::forward<Params>(params)...
-    ).template handle_interruption(std::move(efunc));
+    ).template handle_interruption<OnInterrupt>(std::move(efunc));
```

Comment:
Removing `template` should work, what do you think?
`).handle_interruption(std::move(efunc));`